src/qibolab/platforms/platform.py

```diff
@@ -41,3 +43,33 @@ def sweep(self, sequence, *sweepers, nshots=1024, relaxation_time=None, average=
         return self.design.sweep(
             self.qubits, sequence, *sweepers, nshots=nshots, relaxation_time=relaxation_time, average=average
         )
+
+    def set_lo_drive_frequency(self, qubit, freq):
+        self.qubits[qubit].drive.local_oscillator.frequency = freq
+
+    def get_lo_drive_frequency(self, qubit):
+        return self.qubits[qubit].drive.local_oscillator.frequency
+
+    def set_lo_readout_frequency(self, qubit, freq):
+        self.qubits[qubit].readout.local_oscillator.frequency = freq
+
+    def get_lo_readout_frequency(self, qubit):
+        return self.qubits[qubit].readout.local_oscillator.frequency
+
+    def set_attenuation(self, qubit, att):
+        raise_error(NotImplementedError, f"{self.name} does not support attenuation.")
+
+    def get_attenuation(self, qubit):
+        raise_error(NotImplementedError, f"{self.name} does not support attenuation.")
+
+    def set_gain(self, qubit, gain):
+        raise_error(NotImplementedError, f"{self.name} does not support gain.")
+
+    def get_gain(self, qubit):
+        raise_error(NotImplementedError, f"{self.name} does not support gain.")
+
+    def set_bias(self, qubit, bias):
+        self.qubits[qubit].flux.bias = bias
+
+    def get_bias(self, qubit):
+        return self.qubits[qubit].flux.bias
```

Comment:
Most likely you already know: all of these are really getters and setters, the only reason why you can't simply translate with `@property` is because they are parameterized with `qubit`.

I would like more to move these on the `Qubit` object, such that you can simply do:
```py
platform.qubits[qubit].bias = 5
```
instead of:
```py
platform.set_bias(qubit, 5)
```

If there is no ambiguity, I would also push for:
```py
platform[qubit].bias = 5
```
where you directly access the underlying `qubits` with `__getitem__`.

Reply:
Personally, I totally dislike all these getters/setters. They were introduced in #262 in order to fix the interface with qibocal so I had to add them here to make all routines work with both QM and qblox. The main reason we cannot use something like
```py
platform[qubit].bias = 5
```
right now is that qblox (the `MultiqubitPlatform`) does not use the `Channel` objects (`.bias` is a `Channel` object).

Reply:
Ok, so the solution is just to move to the `DesignPlatform` asap :)